### PR TITLE
Adds conference and google meet support for all new incidents.

### PR DESCRIPTION
**Pull request by kevgliss**
Adds support for automatically creating a video conference on incident creation (default is google meet).

### CODE REVIEW

**Comment by willbengtson** [comment]
+1

File: src/dispatch/conference/service.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return db_session.query(Conference).filter(Conference.resource_id == resource_id).one_or_none()
+
+
+def get_by_resource_type(*, db_session, resource_type: str) -> Optional[Conference]:
+    return (
+        db_session.query(Conference).filter(Conference.resource_type == resource_type).one_or_none()
+    )
+
+
+def get_by_channel_id(db_session, conference_id: str) -> Optional[Conference]:
```
Comment (mvilanova) [inline src/dispatch/conference/service.py]: `get_by_conference_id()`?

File: src/dispatch/conference/service.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    return (
+        db_session.query(Conference).filter(Conference.conference_id == conference_id).one_or_none()
+    )
+
+
+def get_all(*, db_session):
```
Comment (mvilanova) [inline src/dispatch/conference/service.py]: Add return value `List[Optional[Conference]]`

File: src/dispatch/conference/service.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    return db_session.query(Conference)
+
+
+def create(*, db_session, **kwargs) -> Conference:
```
Comment (mvilanova) [inline src/dispatch/conference/service.py]: We should try to use the `ConferenceCreate` model here.

File: src/dispatch/incident/flows.py [hunk 4]
```diff
@@ -413,6 +431,20 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
         "resource_type": INCIDENT_RESOURCE_CONVERSATION_COMMANDS_REFERENCE_DOCUMENT,
     }
 
+    conference = create_conference(incident, [tactical_group["email"]])
+
+    log.debug("Conference created. Tacticle group added.")
```
Comment (mvilanova) [inline src/dispatch/incident/flows.py]: Tactical

File: src/dispatch/incident/flows.py [hunk 5]
```diff
@@ -413,6 +431,20 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
...
+
+    incident.conference = conference_service.create(
```
Comment (mvilanova) [inline src/dispatch/incident/flows.py]: Same comment about using the pydantic model.

File: src/dispatch/conference/service.py (hunk 2 above)
Comment (kevgliss) [reply src/dispatch/conference/service.py]: It returns a query object not a list of objects.

**Pull request merged**